app/src/main/java/mozilla/lockbox/presenter/RoutePresenter.kt

```diff
@@ -30,6 +30,7 @@ import mozilla.lockbox.flux.Presenter
 import mozilla.lockbox.log
 import mozilla.lockbox.store.RouteStore
 import mozilla.lockbox.view.DialogFragment
+import mozilla.lockbox.view.Fragment as SpecializedFragment
```

Comment:
Hmm. I thought it might be clearer to use an obviously-not-android-fragment name, here. Since we're not using `androidx.*.Fragment`, it seems it just adds confusion.